wagtail_content_import/parsers/tables.py

```diff
@@ -0,0 +1,13 @@
+class Table:
+
+    def __init__(self, cells=[]):
+        self.cells = cells
```

Comment:
Probably should be called `rows`

Reply:
Typically it contains all cells though, as a 2D list of lists. Though I suppose the lists inside it are the rows - what do you think? Rename it?